core/src/main/java/de/danoeh/antennapod/core/storage/DBReader.java

```diff
@@ -567,15 +567,19 @@ public static FeedItem getFeedItem(final long itemId) {
     /**
      * Loads a specific FeedItem from the database.
      *
-     * @param podcastUrl the corresponding feed's url
+     * @param guid the corresponding feed's url
      * @param episodeUrl the feed item's url
      * @return The FeedItem or null if the FeedItem could not be found.
      *          Does NOT load additional attributes like feed or queue state.
      */
     @Nullable
-    private static FeedItem getFeedItemByUrl(final String podcastUrl, final String episodeUrl, PodDBAdapter adapter) {
-        Log.d(TAG, "Loading feeditem with podcast url " + podcastUrl + " and episode url " + episodeUrl);
-        try (Cursor cursor = adapter.getFeedItemCursor(podcastUrl, episodeUrl)) {
+    private static FeedItem getFeedItemByGuidOrEpisodeUrl(
+            final String guid,
+            final String episodeUrl,
+            PodDBAdapter adapter
+    ) {
```

Comment:
Please keep the arguments in one line. Only add line breaks if the line is too long (>120 characters). Otherwise, this takes up a lot of vertical screen space.

Reply:
If its not possible to have it in one line I'd rather have it as a list. The 120char limit is arbitrary and the the vertical space issue only arises if you work in landscape mode.
But mostly this ain't flow text. The code has structure that reflects in spacing/indentation and line breaks. Breaking it after an arbitrary 120char limit does not reflect the syntax of the code.